index.d.ts

```diff
@@ -8,6 +8,7 @@ declare namespace Byteball {
     address?: string;
     definition?: any[];
     path?: string;
+    private_key_buff?: object;
```

Comment:
Just "any" should be fine
```suggestion
    private_key_buff?: any;
```

Reply:
Also is it possible to change the param name to `privateKey` instead of `private_key_buff`?

Reply:
I corrected that.